virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc

```diff
@@ -1,14 +1,14 @@
 :_content-type: ASSEMBLY
 [id="virt-using-the-default-pod-network-with-virt"]
-= Using the default pod network for virtual machines
+= Configuring the default pod network
```

Comment:
One thing strikes me as weird. Why on earth would a user need to configure a default network?! You'd think that a default network would be preconfigured. That's the normal definition of "default". I'm going to look into this.

Reply:
I think this whole assembly will change. Shikha is working on a task to move the last module (creating service based on VM) to a more logical place. That just leaves "configuring masquerade mode", which does not exactly require a whole assembly. I think that in the docs reorg we could combine the assemblies:

1. configure masquerade mode for default internal network
2. create bridge, nad, etc. for external networks.

Reply:
haha, excellent point. But yes, that seems logical to me, with my limited networking knowledge.

Reply:
Eureka. I finally figured out what is so weird about this assembly. It's not about Configuring the default pod network at all! It's about configuring the VM to use the default pod network. Keep the file name the same. But you should change the title to something like "Attaching a virtual machine to the default pod network". (Personally, I prefer "Configuring a virtual machine..." because "attaching" sounds like you're using super glue. But that's your call.)

This means that the intro sentences might have to be fixed up a little.

Reply:
> haha, excellent point. But yes, that seems logical to me, with my limited networking knowledge.

I'm sorry -- I didn't realize that your networking knowledge might be limited. It might be more efficient for us to schedule a short meeting next week, outside my office half hour. I could walk you through what is going on. It took me a while to figure things out because all these procedures and modules are, well, a bit verbose!

Reply:
By two separate assemblies do you mean:

- Configuring a virtual machine network interface to use the default network (The new name from above)
- Configuring the default pod network with OpenShift Virtualization

To me these look slightly different though I can see the overlap (network vs. network interface). When you say "Admin guide" I'm sure you're referring to the new organization. Maybe we can work on combining the two assemblies in the future. But if it's something straightforward, it might be do-able. This is just getting confusing for me.

Reply:
We determined at our May 11 meeting we were going to defer this to the general doc reorg project.